npm/src/upload/index.ts

```diff
@@ -71,11 +71,17 @@ export class FileUploader {
     uploadFilesInfo: FileUploadInfo
   ) => {
     window.addEventListener("beforeunload", pageUnloadAction)
+    const currentTimeInSecs = Math.round(new Date().getTime() / 1000)
+    const token =
```

Comment:
I think we can simplify this code and just call the `refreshOrReturnToken` function directly. It's doing essentially the same check as: `this.keycloak.tokenParsed.exp! >= currentTimeInSecs - 5000`